gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureBasePlugin.java

```diff
@@ -93,6 +94,11 @@ private static void createIrTasks(
             compileIr.getProductDependencies().set(project.provider(pdepsExtension::getProductDependencies));
             compileIr.dependsOn(copyConjureSourcesTask);
             compileIr.dependsOn(extractCompilerTask);
+            
+            project.getConfigurations().maybeCreate(CONJURE_IR_CONFIGURATION);
```

Comment:
please use `create`, it should be an error if the user created this configuration already, if the plugin is supposed to own it

Reply:
also need to make it non resolvable:
```java
project.getConfigurations().create(CONJURE_IR_CONFIGURATION, conf -> {
    conf.setCanBeResolved(false);
});
```